### DISCUSSION TITLE
关于bot对进行更新的问题

### DISCUSSION THREAD

**Discussion by CeciliaAki** [Arondight/Adachi-BOT#869] [Q&A]
大佬好！又来叨扰了 TT

在群里查询鹿野院平藏信息的时候，发现bot无反应，猜测可能是没有更新版本，于是按照指引搜索了更新的指令，但似乎没有办法正常更新。代码显示如下

)K2WN5G`01VR7M95XY}%3

应该怎么样才可以正常更新呢？

**Comment by Arondight** [comment]
麻烦截图一下，我不知道你说的错误

**Reply by CeciliaAki** [reply]
![`)K`2WN5G`01VR7M95XY}%3](https://user-images.githubusercontent.com/103917391/182024253-1e31308c-fdf0-4955-962b-68737274bc7f.png)

**Reply by Arondight** [reply]
你这个更新不了，不是 git 仓库，你先克隆一个仓库，然后把当前的文件全部复制过去，然后检出一下不是你修改的文件

**Reply by CeciliaAki** [reply]
很抱歉但想要求更具体的指令，因为不清楚应该如何操作，只在网上搜索到了用git克隆仓库的指令。 OTZ如果可以非常感谢！

**Reply by Arondight** [reply]
```
cd ~/adachibotv3/
git clone https://gitcode.net/iSpeller/Adachi-BOT.git
cd ./Adachi-BOT/
env cp -rvf ../adachibotv3/{config,data,bot.pm2}/ ./
git status
```

随手写的，没验证，大概是这样，具体可能需要微调一下

**Reply by CeciliaAki** [reply]
![~DGM)Q_()@ZXA4U8_U0T$2C](https://user-images.githubusercontent.com/103917391/182025847-9025b7ad-783a-433e-8ebe-c9d5495b6b9d.png)
显示了这样的结果  OAO

**Reply by CeciliaAki** [reply]
![HS_Z~_W($3YTPTAJ %~H09](https://user-images.githubusercontent.com/103917391/182025999-5a76a5f2-5a69-4817-a5eb-c55ca17be30a.png)

**Reply by Arondight** [reply]
```shell
cd ~/adachibotv3/
git clone https://github.com/ixCiel/Adachi-BOT.git
cd ./Adachi-BOT/
env cp -rvf ../Adachi-BOT-myBranch/{config,data,bot.pm2}/ ./
git status
```

**Reply by Arondight** [reply]
你把从 `https://gitcode.net/iSpeller/Adachi-BOT.git` 下载的删除掉，你用的不是我的项目，是 https://github.com/ixCiel/Adachi-BOT.git

**Reply by CeciliaAki** [reply]
好的！感谢！我开始重新下载，但是卡在了9％，可能要花上一段时间TT下载好了后我再回复联系您！
![W~S)2FJSI}LL6FCOKOED(NC](https://user-images.githubusercontent.com/103917391/182026484-0165c905-5594-4a60-9ea6-999cef0555d1.png)

**Reply by Arondight** [reply]
你这个速度得下载到明年

**Reply by CeciliaAki** [reply]
您的锐评一针见血………………|||
我尝试重新下载过很多次但不知为何总是会卡在9％，大约是我这边网络的问题，不可抗力。
实在不行的话我就重新把项目下载一遍吧，非常感谢您的指导！

**Reply by Arondight** [reply]
我已经向项目的所有者说明此事了 https://github.com/ixCiel/Adachi-BOT/issues/11 ，你可以如法炮制，借助 gitcode 来同步项目，然后使用 `git` 命令从同步的镜像拉取

**Reply by CeciliaAki** [reply]
明白了，谢谢您！
